Analysis/DataModel/include/AnalysisDataModel/EventSelection.h

```diff
@@ -44,6 +45,32 @@ DECLARE_SOA_TABLE(EvSels, "AOD", "EVSEL",
                   evsel::SEL8<evsel::BBT0A, evsel::BBT0C, evsel::BBZNA, evsel::BBZNC>,
                   evsel::FoundFT0);
 using EvSel = EvSels::iterator;
+
+// bc-joinable event selection decisions
+namespace bcsel
+{
+// TODO bool arrays are not supported? Storing in int32 for the moment
+DECLARE_SOA_COLUMN(Alias, alias, int32_t[kNaliases]);
+DECLARE_SOA_COLUMN(BBT0A, bbT0A, bool); // beam-beam time in T0A
```

Comment:
There is no need to define the columns again. I propose you put them in the same name space, and then you can reuse them for the second definition